emulators/qemu/Portfile

```diff
@@ -0,0 +1,162 @@
+# $Id$
+
+PortSystem 1.0
+
+name                    qemu
+version                 2.2.0
+categories              emulators
+license                 GPL-2+
+platforms               darwin
+maintainers             raimue openmaintainer
+
+description             Emulator for various architectures
+long_description        QEMU is a generic and open source machine emulator. \
+                        It can run OSes and programs made for one machine on a \
+                        different machine. By using dynamic translation, it \
+                        achieves very good performance.
+
+homepage                http://qemu.org
+master_sites            http://download.qemu.org
+use_bzip2 yes
+
+checksums               rmd160  9a156bdefd8fad72154eee1b10ed53a0e4c1c9a2 \
+                        sha256  b68c9b6c7c694f5489b5a6bffe993cd976ffbb78e7d178eb3bc016caf460039c
+
+patchfiles              patch-configure.diff
+
+depends_build           port:texinfo \
+                        port:libtool \
+                        port:pkgconfig
+
+# python is only used for build scripts, no linking
+depends_build-append    port:python27
```

Comment:
```
set py_ver          2.7
set py_ver_nodot    [string map {. {}} ${py_ver}]
depends_build-append \
                    port:python${py_ver_nodot}
```
(So that if python version is changed, it can be done in one place.)